src/explore-education-statistics-common/src/modules/find-statistics/components/KeyStatTile.tsx

```diff
@@ -31,18 +31,19 @@ const KeyStatTile = ({
   const [dataBlockResponse, setDataBlockResponse] = useState<
```

Comment:
If `dataBlockResponse` is calculated inside of the tile itself, why do we also pass it in as a prop?

Reply:
I assumed there was situation where if the request has been made outside the component, you might want to pass the response in and use that rather than have KeyStatTile make the request.

I think my solution would make this pointless, as now KeyStatTile always makes the request regardless - I wasn't sure how to get around this - but I left it on the off-chance something somewhere was passing down that prop.

I've removed the prop. Everything seems to be still working. Guess I should have just tried to remove it.